### PR TITLE
localize strings

### PR DESCRIPTION
**Pull request by chovy**
## Ticket
https://issues.aphelion-neo.com/tktview?name=fe20c04ef4

## Changes
convert strings to l10n

## Screenshots

screenshot 2018-07-15 13 16 54

## Code Coverage

none

### CODE REVIEW

**Comment by chovy** [comment]
need to verify these all work, so don't merge yet

File: src/renderer/components/dex/OrderHistory.vue [hunk 1]
```diff
@@ -1,22 +1,22 @@
...
-              <th>type</th>
-              <th>pair</th>
-              <th>filled</th>
-              <th>units total</th>
-              <th>price</th>
-              <th>created</th>
-              <th class="status" width="1">status</th>
+              <th>{{$t('order')}}</th>
+              <th>{{$t('type')}}</th>
+              <th>{{$t('pairLc')}}</th>
```
[the remark this answers is not in the document]
Comment (chovy) [reply src/renderer/components/dex/OrderHistory.vue]: I started to change my style halfway through, matching the case.

**Comment by chovy** [comment]
> Lots of changes to account for 275 replacements.

Are you referring to the number of commits?

**Comment by jsolman** [comment]
> Are you referring to the number of commits

No, i'm referring to the number of keys in the en.js file. Quite a few strings in there; nice job.

File: src/renderer/constants.js [hunk 2]
```diff
@@ -7,10 +7,13 @@ const messages = {
...
 
-const langs = [
-  'en',
-  'de',
-];
+const languages = [{
+  label: 'English',
+  value: 'en',
+}, {
+  label: 'German',
```
Comment (jsolman) [inline src/renderer/constants.js]: Oh we need to comment out being able to select German in here until we have all german strings. can you just leave it where english is the only thing in the list for now.

Comment (jsolman) [reply src/renderer/constants.js]: Requesting change for this

**Pull request merged**